job/server/src/main/java/alluxio/job/plan/migrate/MigrateDefinition.java

```diff
@@ -180,10 +185,17 @@ private static void migrate(MigrateCommand command,
           }
           throw t;
         }
+        if (cleanUpRename) {
+          fileSystem.delete(tmpUri);
```

Comment:
@yuzhu I update PR description so hopefully it makes sense now. And do you have better idea about how we can better handle file deletion/rename here?

Reply:
And want to decide the correct process before I update the unit test.